ai_feedback/helpers/constants.py

```diff
@@ -15,5 +11,6 @@
     "output": "Format to display the output response.",
     "test_output": "The output of tests from evaluating the assignment.",
     "submission_image": "The file path for the image file.",
-    "solution_image": "The file path to the solution image."
+    "solution_image": "The file path to the solution image.",
+    "system_prompt": "One of the options defined in args.SystemPrompts for system instructions to the AI Model.",
```

Comment:
This should be more general because the user can specify their own prompts. Follow the same wording as the "prompt" argument above.